spec/list_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'list'
+
+describe List do
+
+  #menu
+
+  it "contains a menu" do
+    list = List.new
+    expect(list.menu).to eq({ "chinese" => 10, "pierogi" => 8, "curry" => 9, "carbonara" => 11, "salad" => 6 })
+  end
+
+  it "responds to the method #show_menu" do
+    list = List.new
+    expect(list).to respond_to(:show_menu)
+  end
+
+  it "shows the menu" do
+    list = List.new
+    expect(list.show_menu).to eq({ "chinese" => 10, "pierogi" => 8, "curry" => 9, "carbonara" => 11, "salad" => 6 })
+  end
+
+  #receipt
+
+  it "contains an receipt of ordered dishes"do
+    list = List.new
+    expect(list.receipt).to eq({})
+  end
+
+  it "responds to method #show_receipt" do
+    list = List.new
+    expect(list).to respond_to(:show_receipt)
+  end
+
+  it "shows receipt" do
+    list = List.new
+    expect(list.show_receipt).to eq(list.receipt)
+  end
+
+
+  #add_dish
+
+  it "responds to the method #add_dish" do
```

Comment:
Please remove the `respond_to` tests once you have written other tests for that method.